src/app/shared/slider-button/slider-button.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Component, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatSliderModule } from '@angular/material/slider';
+import { Metric } from 'src/app/shared/slider-button/enums/metric.enum';
+import { FormControl, FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+
+@Component({
+  selector: 'app-slider-button',
+  standalone: true,
+  imports: [
+    CommonModule,
+    MatSliderModule,
+    ReactiveFormsModule,
+    MatFormFieldModule,
+    MatInputModule,
+    FormsModule
+  ],
+  templateUrl: './slider-button.component.html',
+  styleUrls: ['./slider-button.component.scss']
+})
+export class SliderButtonComponent {
```

Comment:
create a shared module and add this component to it

Reply:
Isn't it better to be standalone and use wherever you want? What are the benefits of having a shared module instead of a standalone component?